distribution/src/root/config/hazelcast-hardened.yaml

```diff
@@ -0,0 +1,233 @@
+    hazelcast.socket.bind.any: false
+
+  management-center:
+    # Explicitly disable the scripting operations sent from the Management center.
+    # The default value is false already so this one is just to be sure and explicit.
+    scripting-enabled: false
+
+  user-code-deployment:
+    # Explicitly disable the user-code deployments. I.e. allow only code which already is on the classpath.
+    # The default value is false already so this one is just to be sure and explicit.
+    enabled: false
+
+  jet:
+    # If you don't need stream/batch processing features, leave them disabled.
+    enabled: false
+    # If you allow the Jet jobs, think twice before allowing custom code upload to the cluster!
+    resource-upload-enabled: false
+
+  # Use the Advanced networking feature to separate protocols used in Hazelcast. Each protocol will have its own endpoint.
+  # Protection introduced by the Advanced networking feature can be simply extended by the operating system features (e.g. firewall).
+  advanced-network:
+    enabled: true
+    join:
+      # Safer than the autodetection is the explicit usage of one discovery method. Let's disable the autodetection.'
+      auto-detection:
+        enabled: false
+      # If you don't need to, don't use the multicast discovery as it advertises own socket address and cluster configuration
+      # within the network
+      multicast:
+        enabled: false
+      # The TCP-IP discovery directly lists explicitly cluster member addresses.
+      tcp-ip:
+        enabled: true
+        # Use the member addresses including the port numbers, so Hazelcast won't try other socket addresses as a fallback.'
+        member-list:
+          - 192.168.1.101:15701
+          - 192.168.1.102:15701
+          - 192.168.1.103:15701
+      aws:
+        enabled: false
+      gcp:
+        enabled: false
+      azure:
+        enabled: false
+      kubernetes:
+        enabled: false
+      eureka:
+        enabled: false
+    # The following part defines different kind of endpoints (protocols) each with its own socket address and configuration.'
+    # Enable only endpoints (protocols) you will need!
+    # For each endpoint only bind to interface with minimal scope access.
+    # The Member endpoint is used for the member-to-member communication. You can increase the endpoint protection
+    # by configuring firewall rules in such a way that only access from other member IP addresses will be allowed.
+    member-server-socket-endpoint-config:
+      interfaces:
+        enabled: true
+        interfaces:
+          - 192.168.1.*
+      port:
+        # use the explicit port number without fallback
+        auto-increment: false
+        # using a non-default port number can prevent an attacker to identify the scanned open port
+        port: 15701
+      # If you have an Enterprise license, you can enable the TLS (SSL) to secure the communication between members.
+      ssl:
+        enabled: false
```

Comment:
why ssl is disabled, because it is EE? I think it would be put a comment about it too